### DISCUSSION TITLE
Uh how do I save

### DISCUSSION THREAD

**Discussion by awccop** [jschuh/klipper-macros#272] [Q&A]
If someone could let me know what command I use to save my printer.cfg after I copy and paste the klipper setup stuff in there I'd appreciate it. I'm using putty if that means anything.

**Comment by kizzelwhix** [comment]
I'd just use the web interface, and hit the 'save & restart' button.

But if you're in nano via ssh, ctrl+x, it'll ask you to save (& to what filename). Just hit enter & enter.
